### PR TITLE
Remove unnecessary config.json keys

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ version: 2
 jobs:
   generic:
     docker:
-      - image: balenalib/amd64-alpine-node:10
+      - image: balenalib/amd64-alpine-node:12
```
Comment (Page-) [inline circle.yml:61]: This is required for the tests to pass since the supervisor requires node 12 now - no idea how this didn't fail on previous PRs

Comment (Page-) [inline circle.yml:61]: Not strictly necessary as it turns out but it makes sense to run the tests on node 12 since that's what we run on

**Comment by Page-** [comment]
@balena-ci rebase

File: circle.yml (hunk 1 above)
Comment (20k-ultra) [reply circle.yml:61]: When I first started using supervisor I had some issues when running on node 10 so I think this is a good change.

**Comment by 20k-ultra** [comment]
Commit is missing Signed-off-by footer. I guess none of the CI jobs caught that?

**Comment by Page-** [comment]
@balena-ci rebase

**Comment by Page-** [comment]
Depends on #1309 to fix the CI build

**Pull request merged**